src/improvement-1.js

```diff
@@ -5,28 +5,18 @@
  */
 
 // FUNCTION 1
-export default function countOfRsInArray(ls) {
```

Comment:
Hey @gitarecy, All looks good to me, despite two things i noticed.

- There is a bit of ambiguity in the naming of your function. 
The function is counting R in a string, not an array. This might lead to confusion, rather than passing a string one would think that it's passing an Array which does not align with the implementation.